sdk/containers/azcontainerregistry/authentication_policy_test.go

```diff
@@ -201,20 +200,11 @@ func Test_authenticationPolicy_anonymousAccess(t *testing.T) {
 	endpoint, _, options := getEndpointCredAndClientOptions(t)
 	client, err := NewClient(endpoint, nil, &ClientOptions{ClientOptions: options})
 	require.NoError(t, err)
-	ctx := context.Background()
 	pager := client.NewListRepositoriesPager(nil)
-	repositoryName := ""
 	for pager.More() {
-		page, err := pager.NextPage(ctx)
+		_, err = pager.NextPage(ctx)
 		require.NoError(t, err)
-		require.NotEmpty(t, page.Repositories.Names)
-		if repositoryName == "" {
-			repositoryName = *page.Repositories.Names[0]
-		}
 	}
-	require.NotEmpty(t, repositoryName)
-	_, err = client.UpdateRepositoryProperties(ctx, repositoryName, &ClientUpdateRepositoryPropertiesOptions{Value: &RepositoryWriteableProperties{CanDelete: to.Ptr(true)}})
-	require.Error(t, err)
```

Comment:
why we remove this? it is aim to ensure anonymous auth could only have the read premission

Reply:
Our client doesn't enforce access restrictions--that's the server's responsibility--so this part of the test was testing the server, not our client.